### PR TITLE
Update for text on multi-exposure dialog

### PR DESCRIPTION
**Pull request by timlinux**
### What does it fix?

@Gustry I updated text for the multi-exposure dialog.

screen shot 2017-11-08 at 15 25 29

### Checklist:
- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

File: safe/gui/ui/multi_exposure_dialog_base.ui [hunk 1]
```diff
@@ -33,7 +33,7 @@
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
-          <string>This tool allows you to run an multi-exposure analysis. By selecting one or many exposures, the impact function will aggregate the results in one report.</string>
+          <string>This tool allows you to run a multi-exposure analysis. Select a hazard, one or more exposures and an aggregation layer. The impact function will aggregate the results of all the hazard / exposure combinations in one report. Use the reporting options tab to specify which layers should be included and visitble in the report. You can set a template on the reporting options tab for producing a printed report. The status tab will show the progress of the analysis as it runs.</string>
```
Comment (Gustry) [inline safe/gui/ui/multi_exposure_dialog_base.ui]: * One exposure only is not allowed, 2 minimum. Should it be?
* s/visitble/visible/g

Comment (timlinux) [reply safe/gui/ui/multi_exposure_dialog_base.ui]: Ok thanks - updated now:

screen shot 2017-11-08 at 15 43 37

**Comment by Gustry** [comment]
Do you want to update the text in the reporting options too?

**Comment by ismailsunni** [comment]
Just popping a suggestion here, after trying the multi exposure dialog. Perhaps we can add a label above the status bar. At first, I thought it was a line or something. Even better if we can show which analysis being run above the status bar. Just UX things.

**Comment by timlinux** [comment]
screen shot 2017-11-09 at 09 06 31

@Gustry reporting options text updated

**Comment by timlinux** [comment]
> Just popping a suggestion here, after trying the multi exposure dialog. Perhaps we can add a label above the status bar. At first, I thought it was a line or something. Even better if we can show which analysis being run above the status bar. Just UX things.

You mean the progress bar right? I think it is mainly a Mac UI issue that it makes skinny progress bars...

**Comment by Gustry** [comment]
LGTM for the help. Thanks @timlinux

**Pull request merged**